src/common/services/session/session.service.spec.js

```diff
@@ -203,11 +203,8 @@ describe('session service', function () {
       it('makes a DELETE request to Cortex & sets postLogoutRedirectUri', done => {
         sessionService
           .signOut(false)
-          .subscribe((response) => {
-            expect(response.data).toEqual({})
-            expect(sessionService.authClient.signOut).toHaveBeenCalledWith({
-              postLogoutRedirectUri: `https://localhost.cru.org:9000/sign-out.html`
-            })
+          .subscribe(() => {
```

Comment:
Can you update the test names as well? Maybe something like "makes a DELETE request to Cortex redirects to Okta" and "should revoke all tokens and redirect to Okta".